tests/test_api.py

```diff
@@ -0,0 +1,11 @@
+from fastapi.testclient import TestClient
+
+from vtpserver.api import app
+
+client = TestClient(app)
+
+
+def test_read_main():
+    response = client.get("/")
+    assert response.status_code == 200
+    assert response.json() == {"message": "Hello World"}
```

Comment:
👍🏼

Reply:
This test file not only tests the API endpoint (with the status code -- nice!) but also demonstrates that our absolute import paths are working.